easybuild/easyconfigs/g/Gaussian/Gaussian-16.A.03-AVX2.eb

```diff
@@ -0,0 +1,62 @@
+easyblock = 'Tarball'
+
+# IMPORTANT!!
+# Needs to be installed with --umask=007
```

Comment:
Why not do a `chmod -R` in `postinstallcmds`?

Reply:
As I said before, 

"The "permissions" step is run after any postinstallcmds and forcibly sets read access to everything that has been installed.

The problem with that is that the gaussian binary starts by checking that it has been installed with "other" unreadable."

Reply:
And it already does the chmod -R in postinstallcmds but it isn't enough for the reason above.

Reply:
And we can't catch this in the sanity check at all? :(

Reply:
Hmm, perhaps... investigating

Reply:
Nope, the permissions step is run AFTER sanity checks, so the files end up with o+r when not using --umask=007